include/pybind11/numpy.h

```diff
@@ -478,7 +480,7 @@ class array : public buffer {
         }
 
         auto tmp = reinterpret_steal<object>(api.PyArray_NewFromDescr_(
-            api.PyArray_Type_, descr.release().ptr(), (int) ndim, shape.data(), strides.data(),
+            api.PyArray_Type_, descr.release().ptr(), (int) ndim, shape.v.data(), strides.v.data(),
```

Comment:
Weird, it's not obvious how it could be ambiguous.

So maybe just `.data()` then? Would we need anything else from it? (data ptr / size / move out into a vec)